src/node/types.h

```diff
@@ -99,6 +102,35 @@ struct BlockCheckOptions {
     bool check_pow{true};
 };
 
+struct CoinbaseTemplate {
+    /* nVersion */
+    uint32_t version;
+    /* nSequence for the only coinbase transaction input */
+    uint32_t input_sequence;
```

Comment:
In commit "mining: add getCoinbase()" (80acabaf9353245f0839b18d9552ab39f595eda6)

Seems like it would be more consistent to either drop the `input_` prefix from this field or add it to `script_sig_prefix` field. I'd slightly prefer dropping the `input_` field so `CoinbaseTemplate` field names describe what they mean, not how the transaction needs to be physically constructed.